hamlpy/test/test_attributes.py

```diff
@@ -106,9 +106,16 @@ def test_read_attribute_dict(self):
         assert dict(self._parse("{class: 'test\u1234'}")) == {'class': 'test\u1234'}
 
         # html style dicts
+        assert dict(self._parse("()")) == {}
+        assert dict(self._parse("(   )")) == {}
+
+        assert dict(self._parse(
+            "(disabled :class='test' data-number = 123\n 'foo'=\"bar\")"
+        )) == {'disabled': None, 'class': 'test', 'data-number': '123', 'foo': 'bar'}
+
         assert dict(self._parse(
-            "(:class='test' :data-number = 123\n foo=\"bar\")"
-        )) == {'class': 'test', 'data-number': '123', 'foo': 'bar'}
+            "(:class='test' data-number = 123\n 'foo'=\"bar\"  \t   disabled)"
```

Comment:
Looks like for {:class} dicts it's the actual Haml syntax, but they're not used for HTML-style dicts http://haml.info/docs/yardoc/file.REFERENCE.html#attributes_

Currently we just ignore them, but seems it should be an error to use one in an HTML-style dict